vllm/transformers_utils/tokenizers/mistral.py

```diff
@@ -291,6 +291,17 @@ def apply_chat_template(self,
 
         from mistral_common.protocol.instruct.request import (
             ChatCompletionRequest)
+
+        # mistral-common requires AssistantMessage content to be string [1].
+        #
+        # [1]: https://github.com/mistralai/mistral-common/blob/f4a06998b75ed78bbf5aaf569590b772ea26c9f6/src/mistral_common/protocol/instruct/messages.py#L80
+        for message in messages:
+            if message.get("role") == "assistant":
+                content = message.get("content")
+                if isinstance(content, list):
+                    content = "\n".join(
+                        [chunk.get("text") for chunk in content])
```

Comment:
```suggestion
                        chunk.get("text") for chunk in content)
```

Nit: Can pass a iterable directly